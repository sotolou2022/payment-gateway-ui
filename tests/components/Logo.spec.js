import { mount, enableAutoUnmount } from "@vue/test-utils";
import { beforeEach, afterEach, expect } from "@jest/globals";
import Component from "~/components/Logo";

let wrapper;

beforeEach(() => {
  wrapper = mount(Component);
});

enableAutoUnmount(afterEach);

describe("Logo component test suite", () => {
  it("is a vue component", () => expect(wrapper.vm).toBeTruthy());
  
  it("match snapshot", () => expect(wrapper.html()).toMatchSnapshot());
});
