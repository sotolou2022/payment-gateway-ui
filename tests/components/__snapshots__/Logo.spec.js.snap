// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component test suite match snapshot 1`] = `"<div class=\\"col text-center my-4\\"><a to=\\"[object Object]\\"><img src=\\"logo.png\\" alt=\\"Logo\\"></a></div>"`;
